src/observer.ts

```diff
@@ -52,6 +57,22 @@ export function observer<P extends object, TRef = {}>(
         memoComponent = memo(wrappedComponent)
     }
 
-    memoComponent.displayName = baseComponentName
+    memoComponent.displayName = `observer(${baseComponentName})`
+    copyStaticProperties(baseComponent as any, memoComponent as any)
```

Comment:
If I am not mistaken, we are manually setting `displayName` above and then again this function? Not a huge deal, but I would probably drop this method and just set the `propTypes` here directly. The comment feels unnecessary as well. Let's just refer to issue #32 in here instead of such essays :D

Reply:
Updated